src/convert.js

```diff
@@ -0,0 +1,27 @@
+import JSBI from 'jsbi'
+
+export function getRawRate(convertRate) {
+  if (typeof convertRate === 'number' || typeof convertRate === 'string') {
+    return convertRate.toString().split('.')
+  }
+  return [convertRate.value, '']
+}
+
+export function getConvertedAmount(amount, convertRate) {
```

Comment:
Like the other parts of the codebase, let's add some JSDoc.

Reply:
Done! https://github.com/aragon/token-amount/pull/7/commits/b4344ce174b62be5994de3fdce8cf18642ba0267